dbcon/joblist/distributedenginecomm.cpp

```diff
@@ -971,6 +973,7 @@ void DistributedEngineComm::write(messageqcpp::ByteStream& msg, uint32_t connect
   lk.unlock();
 
   newClients[connection]->write(msg, NULL, senderStats);
```

Comment:
Is there a way to take newClients[connection]->write() call result into account before we return 0 in the end of DistributedEngineComm::write()?